spring-cloud-alibaba-examples/integrated-example/helm-chart/templates/integrated-nacos-mysql.yaml

```diff
@@ -0,0 +1,57 @@
+apiVersion: v1
+kind: ReplicationController
+metadata:
+  name: nacos-mysql
+  labels:
+    name: nacos-mysql
+spec:
+  replicas: 1
+  selector:
+    name: nacos-mysql
+  template:
+    metadata:
+      labels:
+        name: nacos-mysql
+    spec:
+      containers:
+        - name: mysql
+          image: trevorlink/nacos-mysql
```

Comment:
There is a mistake here that needs to be corrected. BTW, I didn't see the pod when I deployed the whole demo and please check whether it is a valuable content?